src/gap.c

```diff
@@ -2814,128 +2814,51 @@ void ThreadedInterpreter(void *funcargs) {
-    { "LOAD_DYN", 2, "filename, crc",
-      FuncLOAD_DYN, "src/gap.c:LOAD_DYN" },
-
-    { "LOAD_STAT", 2, "filename, crc",
-      FuncLOAD_STAT, "src/gap.c:LOAD_STAT" },
-
-    { "SHOW_STAT", 0, "",
-      FuncSHOW_STAT, "src/gap.c:SHOW_STAT" },
-
-    { "GASMAN", -1, "args",
-      FuncGASMAN, "src/gap.c:GASMAN" },
-
-    { "GASMAN_STATS", 0, "",
-      FuncGASMAN_STATS, "src/gap.c:GASMAN_STATS" },
-
-    { "GASMAN_MESSAGE_STATUS", 0, "",
-      FuncGASMAN_MESSAGE_STATUS, "src/gap.c:GASMAN_MESSAGE_STATUS" },
-
-    { "GASMAN_LIMITS", 0, "",
-      FuncGASMAN_LIMITS, "src/gap.c:GASMAN_LIMITS" },
-
-    { "SHALLOW_SIZE", 1, "object",
-      FuncSHALLOW_SIZE, "src/gap.c:SHALLOW_SIZE" },
-
-    { "TNUM_OBJ", 1, "object",
-      FuncTNUM_OBJ, "src/gap.c:TNUM_OBJ" },
-
-    { "TNUM_OBJ_INT", 1, "object",
-      FuncTNUM_OBJ_INT, "src/gap.c:TNUM_OBJ_INT" },
-
-    { "OBJ_HANDLE", 1, "object",
-      FuncOBJ_HANDLE, "src/gap.c:OBJ_HANDLE" },
-
-    { "HANDLE_OBJ", 1, "object",
-      FuncHANDLE_OBJ, "src/gap.c:HANDLE_OBJ" },
-
-    { "LoadedModules", 0, "",
-      FuncLoadedModules, "src/gap.c:LoadedModules" },
-
-    { "WindowCmd", 1, "arg-list",
-      FuncWindowCmd, "src/gap.c:WindowCmd" },
-
-    { "MicroSleep", 1, "msecs",
-      FuncMicroSleep, "src/gap.c:MicroSleep" },
-
-    { "Sleep", 1, "secs",
-      FuncSleep, "src/gap.c:Sleep" },
-
-    { "GAP_EXIT_CODE", 1, "exit code",
-      FuncGAP_EXIT_CODE, "src/gap.c:GAP_EXIT_CODE" },
-
-    { "QUIT_GAP", -1, "args",
-      FuncQUIT_GAP, "src/gap.c:QUIT_GAP" },
-
-    { "FORCE_QUIT_GAP", -1, "args",
-      FuncFORCE_QUIT_GAP, "src/gap.c:FORCE_QUIT_GAP" },
-
+    GVAR_FUNC(Runtime, 0, ""),
+    GVAR_FUNC(RUNTIMES, 0, ""),
+    GVAR_FUNC(NanosecondsSinceEpoch, 0, ""),
+    GVAR_FUNC(NanosecondsSinceEpochInfo, 0, ""),
+    GVAR_FUNC(SizeScreen, -1, "args"),
+    GVAR_FUNC(ID_FUNC, 1, "object"),
+    GVAR_FUNC(RETURN_FIRST, -1, "object"),
+    GVAR_FUNC(RETURN_NOTHING, -1, "object"),
+    GVAR_FUNC(ExportToKernelFinished, 0, ""),
+    GVAR_FUNC(DownEnv, -1, "args"),
+    GVAR_FUNC(UpEnv, -1, "args"),
+    GVAR_FUNC(GAP_CRC, 1, "filename"),
+    GVAR_FUNC(LOAD_DYN, 2, "filename, crc"),
+    GVAR_FUNC(LOAD_STAT, 2, "filename, crc"),
+    GVAR_FUNC(SHOW_STAT, 0, ""),
+    GVAR_FUNC(GASMAN, -1, "args"),
+    GVAR_FUNC(GASMAN_STATS, 0, ""),
+    GVAR_FUNC(GASMAN_MESSAGE_STATUS, 0, ""),
+    GVAR_FUNC(GASMAN_LIMITS, 0, ""),
+    GVAR_FUNC(SHALLOW_SIZE, 1, "object"),
+    GVAR_FUNC(TNUM_OBJ, 1, "object"),
+    GVAR_FUNC(TNUM_OBJ_INT, 1, "object"),
+    GVAR_FUNC(OBJ_HANDLE, 1, "object"),
+    GVAR_FUNC(HANDLE_OBJ, 1, "object"),
+    GVAR_FUNC(LoadedModules, 0, ""),
+    GVAR_FUNC(WindowCmd, 1, "arg-list"),
+    GVAR_FUNC(MicroSleep, 1, "msecs"),
+    GVAR_FUNC(Sleep, 1, "secs"),
+    GVAR_FUNC(GAP_EXIT_CODE, 1, "exit code"),
+    GVAR_FUNC(QUIT_GAP, -1, "args"),
+    GVAR_FUNC(FORCE_QUIT_GAP, -1, "args"),
     { "SHOULD_QUIT_ON_BREAK", 0, "",
       FuncShouldQuitOnBreak, "src/gap.c:FuncShouldQuitOnBreak"},
```

Comment:
For examples like this, I'd consider renaming the C function to match the GAP one. OK?